helm/sematic-server/values.yaml

```diff
@@ -21,6 +18,10 @@ database:
   url: postgres://postgresql:5432/postgres # REPLACE ME
 
 deployment:
+  socket_io:
+    dedicated: false
+    address: https://my.sematic-socketio # REPLACE ME
```

Comment:
what should this be replaced with? Shouldn't we be able to infer it, given that we are creating the service?

Reply:
Adopted the same solution as for the default Server service.
Removed the configuration and using the auto-constructed service name.